### PR TITLE
default statement timeout set to 5 minutes

### PR DESCRIPTION
**Pull request by sergeikiktev**
#SG-2427

Please check

### CODE REVIEW

File: setup/postgresql_bootstrap_centos7.sh [hunk 1]
```diff
@@ -110,6 +110,9 @@ END_OF_HBA_CONF"
 
   # Log message is produced when a session waits longer than deadlock_timeout to acquire a lock.
   sudo sed -i -e "s/.*log_lock_waits.*/log_lock_waits = on/" ${PGDATA}/postgresql.conf
+  
+  # Set a default statement timeout to 5 minutes
+  sudo sed -i -e "s/.*statement_timeout.*/statement_timeout = 5min/" ${PGDATA}/postgresql.conf
```
Comment (david-richer-adsk) [inline setup/postgresql_bootstrap_centos7.sh:115]: should be `= '5min'`

Quotes are required when using units: https://www.postgresql.org/docs/9.6/static/config-setting.html

Comment (sergeikiktev) [reply setup/postgresql_bootstrap_centos7.sh:115]: Interesting. They probably do not follow their own requirements. :-)
screen shot 2018-03-07 at 19 04 03
screen shot 2018-03-07 at 18 59 25

Comment (david-richer-adsk) [reply setup/postgresql_bootstrap_centos7.sh:115]: doc is probably to up-to-date, feel free to commit as is.

**Comment by sergeikiktev** [comment]
screen shot 2018-03-08 at 13 06 42

Can you try to merge this pull request?

**Comment by david-richer-adsk** [comment]
did the merge

**Pull request merged**